recipes/03-frontend-assets.adoc

```diff
@@ -47,13 +47,20 @@ module.exports = {
 }
 ----
 
-Run `./node_modules/.bin/webpack` to build your files. Also `--watch` flag can be passed to start a watcher.
+Run `./node_modules/.bin/webpack` to build your files. Use the flag `--mode` to choose between production and development.
```

Comment:
Can you convert the flags doc to `list items`. So it will be

```
- Use the flag `--mode` to choose between production and development.
- To start the watcher, make use of `--watch` flag.
```